pkg/jobrunaggregator/jobrunaggregatorlib/jobrun_locator_release_jobs.go

```diff
@@ -9,12 +9,19 @@ import (
 	prowjobv1 "k8s.io/test-infra/prow/apis/prowjobs/v1"
 )
 
-func GetPayloadTagFromProwJob(prowJob *prowjobv1.ProwJob) string {
+// GetPayloadTagFromProwJobForAggregatedJob gets the payload tag from prow jobs for aggregated jobs.
+func GetPayloadTagFromProwJobForAggregatedJob(prowJob *prowjobv1.ProwJob) string {
 	return prowJob.Labels["release.openshift.io/analysis"]
 }
 
+// GetPayloadTagFromProwJobForNonAggregatedJob gets the payload tag from prow jobs for non-aggregated jobs.
+func GetPayloadTagFromProwJobForNonAggregatedJob(prowJob *prowjobv1.ProwJob) string {
+	return prowJob.Annotations["release.openshift.io/tag"]
```

Comment:
does this tag exist on all release controller created prowjobs?  Do we need to use "release.openshift.io/analysis" for some reason?

Reply:
Yes, we need to use "release.openshift.io/analysis" just to get the aggregated jobs. Otherwise we will be aggregating the informing jobs as well.

Reply:
> Yes, we need to use "release.openshift.io/analysis" to get just the aggregated jobs. Otherwise we will be aggregating the informing jobs as well.

I thought we limited by job type.

Reply:
I am still going to differentiate between aggregated vs non-aggregated in this PR. If we want to use "release.openshift.io/tag" for all jobs, I will test the path in a separate PR.

Reply:
Items added to lookup table. So we can use a single payload tag label now.